setup.py

```diff
@@ -66,11 +66,13 @@
 XPALIB_DIR = "cextern/xpa-2.1.15"
 CONF_H_NAME = os.path.join(XPALIB_DIR, "conf.h")
 
+# Output from Cythonizing
+CYTHON_SOURCE = "wrappers/xpa.c"
+
 # check if cython is available.
 try:
     from Cython.Build import cythonize
     cythonize("wrappers/xpa.pyx")
-    CYTHON_SOURCE = "wrappers/xpa.c"
     have_cython=True
 except ImportError:
     print("Unable to load Cython")
```

Comment:
But is Cython somehow needed to *run* the code or just build/install it?  

Typically what we do is set things up so Cython is only needed for build, and only from git at that--source tarballs are released with the generated C sources already included, so that someone installing via pip, say, does not need Cython.